src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -31,23 +31,21 @@ public static class IntakeConstants {
 		public static final double INTAKE_IN_SPEED = 0.33;
 		public static final double INTAKE_OUT_SPEED = -0.05;
 
-		public static final int INTAKE_COLOR_THRESHOLD = 10;
 		public static final double LED_PURPLE = 0.91;
 		public static final double LED_YELLOW = 0.69;
+		public static final double LED_OFF = 0.99;
 
 		// enums
 		public static enum GamePieceType {
```

Comment:
Do we still need GamePieceType? Looking at the references, I see some unused imports and the distance in `isSecured()`, which could easily be changed to a single constant.